lib/matplotlib/lines.py

```diff
@@ -45,13 +45,9 @@ def _get_dash_pattern(style):
     elif isinstance(style, tuple):
         offset, dashes = style
         if offset is None:
-            _api.warn_deprecated(
-                "3.3", message="Passing the dash offset as None is deprecated "
```

Comment:
AFAICS there is no removal entry for this.

Reply:
Yes, that was missed; added https://github.com/matplotlib/matplotlib/pull/20465/files#diff-614da5f81650b40c2bf17da3ca07deb7971a039cb29ad5e6850ef52167cb9e4dR38-R39